Utils/insertBoxesToDB.js

```diff
@@ -0,0 +1,30 @@
+// imports xlsx file as an array
+// loop over the array imported
+// take the relevant data to a single box
+// run query of inserting box with the data
+
+const importXlsxFile = require('./importXlsxFile');
+const insertBoxes = require('../server/database/queries/insertBoxes');
+
+const path = '/home/ray261/Desktop/BlackSheet/MINI_BOXES.xlsx';
```

Comment:
This is a relative path to a file on your desktop, you should add the file to this repo for this to work for everyone

Reply:
We didn't want to add the file to the public repo because it contains sensitive/private data.
Anyway I talked with @Ryan2601 he will set the path to be taken as an arg env variable since this file is called using through node command shell.

Reply:
Cool, if that file can be found online then I'm not sure it's that sensitive :ok_woman:, but happy for it to be in an env file.
reassign me when it's done